content/docs/command-reference/exp/show.md

```diff
@@ -109,7 +109,9 @@ metric or param.
 - `--sort-by <name>` - sort experiments by the specified metric or param
   (`name`). Only one visible column (either metric or param) can be used for
   sorting. This only affects the ordering of experiments derived from the same
-  parent commit. Parent commits are always sorted chronologically.
+  parent commit. Parent commits are always sorted chronologically. When combined
+  with the `--html` flag, `name` will be used to set the color of the lines in
```

Comment:
This "sets the color" expression is a bit confusing. Could you clarify how the color is determined?